src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala

```diff
@@ -133,6 +137,13 @@ class EasyConvertBagToDepositApp(configuration: Configuration) extends DebugEnha
       Failure(e)
   }
 
+  private def addToXmlFile(filesXmlFile: String, filesToAdd: Seq[String]): Try[File] = Try {
+    val oldFilesXml = XML.loadFile(filesXmlFile)
+    val newFilesXml = FilesXml(oldFilesXml, "data/easy-migration", filesToAdd, "text/xml")
+    File(filesXmlFile).delete()
+    File(filesXmlFile).writeText(printer.format(newFilesXml))
```

Comment:
PrettyPrinter can be nice for Unit tests, please use https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/23f0eb93dbc2cfa9cddd78904a0c5b9a1f63eede/src/main/scala/nl.knaw.dans.easy.bag2deposit/package.scala#L69-L72 as done for ddm and provenance.
Sadly I did not mention the reason in 476fadd2 but it was something that had to be changed in the whole chain of every application that wrote XML's.

Reply:
The reason is mentioned in https://github.com/DANS-KNAW/dans-development-team-standup/blob/47f7ad73e1cd7553fd21097841186f2cef75b102/2021-04-06.md#L60
PrettyPrinter reduces white space. Hence paragraphs and new lines dissappeared from ddm/emd descriptions.

Reply:
without PrettyPrinter (and with .serialize) `files.xml` looks like:

    ...
    <file filepath="data/images/PAN-00077978-002.jpg">
        <dcterms:format>image/jpeg</dcterms:format>
    </file>

            <file filepath="data/easy-migration/provenance.xml">
           <dcterms:format>text/xml</dcterms:format>
        </file><file filepath="data/easy-migration/emd.xml">
           <dcterms:format>text/xml</dcterms:format>
        </file><file filepath="data/easy-migration/dataset.xml">
           <dcterms:format>text/xml</dcterms:format>
        </file><file filepath="data/easy-migration/files.xml">
           <dcterms:format>text/xml</dcterms:format>
        </file>
          </files>

Would the possible white space reducing be a problem in `files.xml`?

Reply:
Perhaps not, but I prefer to do the same things in the same way. Otherwise someone might get tempted to refactor the other occasions to PrettyPrinter and reintroduce old problems.

Reply:
just using serialize will leave the xml-file really messy.  I have added a comment, explaining why PrettyPrinter is used here, but discouraging its use in general.

Reply:
@lindareijnhoudt 
I vaguely remember that files can have additional custom metadata. Can't these suffer from changes in white space in free text fields?

Reply:
If this is really a potential problem, I think we should make a method that lines up the new elements  neatly, without deleting white space inside the elements. If you think it is worth it, I could make it.